rust/scx_utils/src/gpu.rs

```diff
@@ -118,12 +118,18 @@ pub fn create_gpus() -> BTreeMap<usize, Vec<Gpu>> {
                 .performance_state()
                 .unwrap_or(PerformanceState::Unknown);
 
-            // The NVML library doesn't return a PCIe bus ID compatible with sysfs. It includes
-            // uppercase bus ID values and an extra four leading 0s.
-            let bus_id = pci_info.bus_id.to_lowercase();
-            let fixed_bus_id = bus_id.strip_prefix("0000").unwrap_or("");
-            let numa_path = format!("/sys/bus/pci/devices/{}/numa_node", fixed_bus_id);
-            let numa_node = read_from_file(Path::new(&numa_path)).unwrap_or(0_usize);
+            let numa_node = device
```

Comment:
This doesn't look right:
```
 cargo build --release --features gpu-topology -p scx_utils
warning: /home/arighi/src/scx/rust/scx_rustland_core/Cargo.toml: unused manifest key: lib.include
   Compiling scx_utils v1.0.24 (/home/arighi/src/scx/rust/scx_utils)
error[E0425]: cannot find value `device` in this scope
   --> rust/scx_utils/src/gpu.rs:121:29
    |
121 |             let numa_node = device
    |                             ^^^^^^ not found in this scope
```

Maybe `numa_gpu`? But it doesn't seem to have the `numa_node_id()` method.